install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -17,6 +17,6 @@ data:
       "baremetalIronic": "quay.io/openshift/origin-ironic:v4.2.0",
       "baremetalIronicInspector": "quay.io/openshift/origin-ironic-inspector:v4.2.0",
       "baremetalIpaDownloader": "quay.io/openshift/origin-ironic-ipa-downloader:v4.2.0",
-      "baremetalRhcosDownloader": "quay.io/openshift/origin-ironic-rhcos-downloader:v4.2.0",
+      "baremetalMachineOsDownloader": "quay.io/openshift/origin-ironic-machine-os-downloader:v4.3.0",
```

Comment:
has this image gone through ART build process?

Reply:
I believe so, @elfosardo?

Reply:
the image was just renamed, this is not a new image, so yes :)

Reply:
ART is building it as `ironic-machine-os-downloader`, yes.

Reply:
Thanks. Resolving this comment.